docs/userguide.md

```diff
@@ -687,41 +687,41 @@ Resources such as `namespaces` associated with a `FederatedNamespace` or `Federa
 should be deleted before cleaning up the deployment, otherwise, the process will fail.
 
 Run the following command to perform a cleanup of the cluster registry and
-kubefed deployments:
+KubeFed deployments:
 
 ```bash
 ./scripts/delete-federation.sh
 ```
 
-The above script unjoins the all of the clusters from the kubefed control plane it deploys,
+The above script unjoins the all of the clusters from the KubeFed control plane it deploys,
 by default.
 
 On successful completion of the script, both `cluster1` and
-`cluster2` will be unjoined from the deployed kubefed control plane.
+`cluster2` will be unjoined from the deployed KubeFed control plane.
 
-## Namespaced Federation
+## Namespace-scoped control plane
 
 All prior instructions referred to the deployment and use of a
-cluster-scoped kubefed control plane. It is also possible to
+cluster-scoped KubeFed control plane. It is also possible to
 deploy a namespace-scoped control plane. In this mode of operation,
-kubefed controllers will target resources in a single namespace on
+KubeFed controllers will target resources in a single namespace on
 both host and member clusters. This may be desirable when
-experimenting with kubefed on a production cluster.
+experimenting with KubeFed on a production cluster.
 
 ### Helm Configuration
 
-To deploy a federation in a namespaced configuration, set
+To deploy KubeFed in a namespaced configuration, set
 `global.scope` to `Namespaced` as per the Helm chart [install
 instructions](https://github.com/kubernetes-sigs/kubefed/blob/master/charts/kubefed/README.md#configuration).
 
 
 ### Joining Clusters
 
-Joining additional clusters to a namespaced federation requires
+Joining additional clusters to a namespaced deployment requires
```

Comment:
s/deployment/control plane/

Reply:
Not sure what you mean by this? Change deployment to control plane?

Reply:
Yes, sorry.  `s/<original text>/<replacement text/` is programmer shorthand for changing text.

In this instance 'deployment' is ambiguous, since it can refer to a 'Deployment' (i.e. a kubernetes resource) or a deployment of KubeFed (i.e. control plane).